mobile/status.go

```diff
@@ -746,15 +745,3 @@ func SignHash(hexEncodedHash string) string {
 
 	return hexEncodedSignature
 }
-
-func GenerateAlias(pk string) string {
```

Comment:
@adambabik Looks like you deleted these methods by mistake?

Reply:
Yeah, you're missing 9a3ed980c9dcbc6e55fcba843224e447b06eb08f, and probably also 40e66e6186164a422e432fd754b8f86b958014f3.

Reply:
Oh whoops. I rebased against develop and resolved all conflicts using `-s ours`... I should use that only on `vendor/` conflicts. Fixing.